nbs/tsdataset.ipynb

```diff
@@ -54,11 +54,12 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #81.            min_size = int('inf')
</pre><p>I think float('inf') as fine here, I get an error when trying to run this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>